editor/src/components/inspector/controls/color-picker.tsx

```diff
@@ -37,6 +44,17 @@ export const ColorPicker: React.FunctionComponent<React.PropsWithChildren<ColorP
   portalTarget,
   ...props
 }) => {
+  const onSubmitValue = props.onSubmitValue
+  const dispatchEyeDropper = React.useCallback(() => {
+    const EyeDropper = window.EyeDropper
+    if (EyeDropper == null) {
+      return
+    }
+    closePopup()
+    void new EyeDropper().open().then((result: any) => {
```

Comment:
I dislike this `any`. Assuming there is no way we can guarantee type safety here, can we instead fall back to using `unknown` and check that is is an object with a `sRGBHex` key of type string before trying to then use that value. Even if we fail silently or write a console error if not.

Reply:
absolutely! fixed in https://github.com/concrete-utopia/utopia/pull/3115/commits/79100a9a4b198caa19167298c517d86bccf8c5df